Libraries/Components/AccessibilityInfo/NativeAccessibilityInfo.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+
+export interface Spec extends TurboModule {
+  +isReduceMotionEnabled: (
+    resolve: (isReduceMotionEnabled: boolean) => void,
+  ) => void;
+  +isTouchExplorationEnabled: (
+    resolve: (isScreenReaderEnabled: boolean) => void,
+  ) => void;
+  +setAccessibilityFocus: (reactTag: number) => void;
+  +announceForAccessibility: (announcement: string) => void;
+
+  // RCTDeviceEventEmitter
+  +addListener: (eventName: string, handler: Function) => Object;
+  +removeListeners: (eventName: string, handler: Function) => void;
```

Comment:
NativeModules on Android don't expose inherited methods to JS. And I don't see these methods in `AccessibilityInfoMethod`. Are you sure these methods belong here?

Reply:
I wasn't sure about that, I'm going to remove it.